src/DSL.ts

```diff
@@ -0,0 +1,20 @@
+import { queryStrict } from './Query';
+import { taskEither } from 'fp-ts/lib/TaskEither';
+import { refetch } from './Strategy';
+
+export function param<A>() {
+  return queryStrict((a: A) => taskEither.of<void, A>(a), refetch);
+}
+
+export { queryShallow, queryStrict, queryJSON } from './Query';
+
+export { available, refetch, expire } from './Strategy';
+
+export { compose, product } from './Query';
+
+import { observeShallow } from './observe';
+
+export const observe = observeShallow;
```

Comment:
why are we exporting this?

Reply:
this deserves an offline discussion I believe, good question 👍